modules/vaos/app/services/eps/appointment_service.rb

```diff
@@ -10,6 +12,13 @@ class AppointmentService < BaseService
     def get_appointments
       response = perform(:get, "/#{config.base_path}/appointments?patientId=#{patient_id}",
                          {}, headers)
+
+      track_metric(STATSD_KEY, 'get_appointments call completed')
```

Comment:
Could you please see if we can make use of existing [vaos_logging](https://github.com/department-of-veterans-affairs/vets-api/blob/3b7435f8b9bb33bb781ea018716522f38bf75dd2/modules/vaos/app/services/vaos/middleware/vaos_logging.rb#L24-L24) middleware?